Makefile

```diff
@@ -360,6 +374,34 @@ azure-ipam-image-pull: ## pull azure-ipam container image.
 		IMAGE=$(AZURE_IPAM_IMAGE) \
 		TAG=$(AZURE_IPAM_PLATFORM_TAG)
 
+# ipv6-healthprobe-tc
+
+ipv6-healthprobe-tc-image-name: # util target to print the ipv6-healthprobe-tc image name.
```

Comment:
instead of healthprobe can we name as hp. something like `ipv6-hp-bpf`? and do it everywhere

Reply:
ok, updated